biggie and kas2c will fail with current script setup on macOS

This is what I get when trying to run these tools:

1. biggie
    `cd ./tools/biggie && sh ./biggie-MacOSX-compile.sh`:

    ```sh
    In file included from ../../src/Game/Types.c:5:
    ../../src/Game/Types.h:15:10: fatal error: 'SDL_stdinc.h' file not found
    #include "SDL_stdinc.h"
             ^~~~~~~~~~~~~~
    1 error generated.
    In file included from ../../src/Game/BigFile.c:18:
    In file included from ../../src/Game/BigFile.h:12:
    ../../src/Game/Types.h:15:10: fatal error: 'SDL_stdinc.h' file not found
    #include "SDL_stdinc.h"
             ^~~~~~~~~~~~~~
    1 error generated.
    In file included from main.c:12:
    In file included from ./../../src/Game/BigFile.h:12:
    ./../../src/Game/Types.h:15:10: fatal error: 'SDL_stdinc.h' file not found
    #include "SDL_stdinc.h"
         ^~~~~~~~~~~~~~
    1 error generated.
    ```

2. kas2c
    `cd ./tools/kas2c && sh ./kas2c-MacOSX-compile.sh`

    ```sh
    In file included from lexer.l:8:
    ./../../src/Game/Types.h:15:10: fatal error: 'SDL_stdinc.h' file not found
    #include "SDL_stdinc.h"
            ^~~~~~~~~~~~~~
    1 error generated.
    In file included from parser.y:18:
    ./../../src/Game/Types.h:15:10: fatal error: 'SDL_stdinc.h' file not found
    #include "SDL_stdinc.h"
            ^~~~~~~~~~~~~~
    1 error generated.
    ```

This doesn't make sense considering that the SDL lib points to the local Xcode workspace folder (602a016): https://github.com/KajiTetsushi/HomeworldSDL/blob/56ee4dc544260811748ec7e298aa28abf565d0b4/.gitignore#L5-L8

--- a/tools/biggie/biggie-MacOSX-compile.sh
+++ b/tools/biggie/biggie-MacOSX-compile.sh
@@ -4,7 +4,7 @@
 
 /usr/bin/gcc                                       \
     -D _MACOSX                                     \
-    -I /Library/Frameworks/SDL2.framework/Headers  \
+    -I ../../Mac/SDL2.framework/Headers            \
     -I ../../src/SDL                               \
     -I ../../src/ThirdParty/CRC                    \
     -I ../../src/ThirdParty/LZSS                   \

--- a/tools/kas2c/kas2c-MacOSX-compile.sh
+++ b/tools/kas2c/kas2c-MacOSX-compile.sh
@@ -11,7 +11,7 @@
 
 /usr/bin/gcc                                       \
     -D_MACOSX                                      \
-    -I /Library/Frameworks/SDL2.framework/Headers   \
+    -I ../../Mac/SDL2.framework/Headers            \
     *.c                                            \
     -o kas2c
 

